src/cantools/database/can/formats/kcd.py

```diff
@@ -419,7 +419,8 @@ def _dump_message(message, bus, node_refs, sort_signals):
 
 
 def _dump_version(version, parent):
-    SubElement(parent, 'Document', version=version)
+    if version is not None:
+        SubElement(parent, 'Document', version=version)
```

Comment:
Wouldn't it be better to set version to an emtpy string?
```suggestion
        SubElement(parent, 'Document', version=version or "")
```

Reply:
I couldn't find an exact definition of how the [version field](https://github.com/dschanoeh/Kayak/blob/ddf5f9fbee7b96e596d83bbab24dfa74edc39a63/Kayak-kcd/src/main/resources/com/github/kayak/canio/kcd/loader/Definition.xsd#L426) should be represented so I thought that skipping it would have been better than setting an unexpected value, and since [`load_string`](https://github.com/cantools/cantools/blob/116333d1e1a01dc3afa3f03c5b646a6d512392af/src/cantools/database/can/formats/kcd.py#L478) sets it to `None` if not found I kept the same standard. Let me know which way you prefer.

Reply:
do the other tools which support KCD work if the element is not present? (cf. https://github.com/julietkilo/kcd?tab=readme-ov-file#projects)

Reply:
@andlaus
+ **can4python** ignores the root metadata [[1]][c4p_kcd] when reading a KCD db and does not set it at all [[2]][c4p_constants] when writing it; tested with `can.CanBus.from_kcd_file('test.kcd', 'vcan0', ego_node_ids=["1"])`.
+ The same goes for **canmatrix** ([[1]][cm_kcd_write], [[2]][cm_kcd_read]); tested with `canmatrix.formats.kcd.load('test.kcd')`.

I have not tested the Java and C++ tools due to missing or unclear examples/docs/build environments.

[c4p_kcd]: https://github.com/caran/can4python/blob/4177bcf4368fd8484ec080e2d926cb999b14c6a5/can4python/filehandler_kcd.py
[c4p_constants]: https://github.com/caran/can4python/blob/4177bcf4368fd8484ec080e2d926cb999b14c6a5/can4python/constants.py#L68
[cm_kcd_write]: https://github.com/ebroecker/canmatrix/blob/561eccc9c8465bddda0686750fd0af76ec4c8f6f/src/canmatrix/formats/kcd.py#L136
[cm_kcd_read]: https://github.com/ebroecker/canmatrix/blob/561eccc9c8465bddda0686750fd0af76ec4c8f6f/src/canmatrix/formats/kcd.py#L343